syntax = "proto3";

message PersonMessage {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

message Empty {

}

message PeopleMessageList {
  repeated PersonMessage people = 1;
}

service PeopleService {
  rpc Create(PersonMessage) returns (PersonMessage);
  rpc Get(Empty) returns (PeopleMessageList);
}
